src/ActionBar/context.tsx

```diff
@@ -21,8 +22,15 @@ export const useActionBar = () => {
 export const ActionBarProvider: React.FC = ({ children }) => {
   const anchor = React.useRef<HTMLDivElement | null>(null);
 
+  // Force re-render after ref is updated
+  const [, updateState] = React.useState(0);
```

Comment:
Its possible to set ref not only by using `useRef`, but `useState` too. Can you check if it would work?